index.js

```diff
@@ -0,0 +1,116 @@
+
+//player attributes
+const playerName = document.querySelector("h2")
+const playerImage = document.querySelector(".player img")
+const playerLikes = document.querySelector(".player p")
+const playerNickname = document.querySelector("em")
+//added player number to html
+const playerNumber = document.querySelector("#player-number")
+
+//page attributes
+const ul = document.querySelector("#goals")
+const liker = document.querySelector(".like-button")
+const videoForm = document.querySelector("#new-goal-form")
+
+
+//make network request to API
+fetch("http://localhost:3000/players/1")
+  //promise 1
+  .then(response => response.json())
+  //promise 2
+  .then(playerObj => {
+    //use data response from API
+    playerImage.src = playerObj.photo
+    playerImage.alt = playerObj.name 
+    playerName.textContent = playerObj.name 
+    playerNickname.textContent = playerObj.nickname
+    playerNumber.textContent = `Player Number: ${playerObj.number}`
+    playerImage.parentNode.dataset.id = playerObj.id 
+    playerLikes.textContent = `${playerObj.likes} Likes`
+    playerObj.goals.forEach(goal => {
+          renderGoal(goal)
+    })
+  })
+
+//Goal Video Submit 
+videoForm.addEventListener("submit", function(event) {
+  event.preventDefault()
+  const newGoal = {
+      //got stuck here - keep getting "null" for player-id :(
+        //never mind figured out that player is a sibling element NOT a parent!!!
+      playerId: event.target.previousElementSibling.getAttribute("data-id"),
```

Comment:
Here, it may have been easier to just add the `data-id` directly to the form element instead, so you could just do `event.target.dataset.id`. You'll also need to `parseInt` the number - the `playerId` **must** be a number in order for the server to associate the goal with the player.